### PR TITLE
Make website dynamic using JavaScript

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
# New Feature 🆕
- ✅ Add new JavaScript file `pop-up.js` to perform the requirements for this milestone.
- ✅ Add **EventListener** to the project buttons `See Project`
- ✅ Make the website more dynamic
  - ✔For every single _see project_ button the technologies on project descriptions change.
  - ✔On the pop-up window there are two buttons: **See Live** and **See Source** that navigate to the related pages in a new tab.-
- ✅ No linters error.
- ✅ Figma [template]() is preserved as much as possible.
### To the code reviewer 👨‍💻 
- [ ] _I kindly ask dear code reviewer that if there is any kind of issue in this project, please do list them in a descriptive manner and give your best suggestions if needed._
- [ ] _If you think there is some big issues that is essential to be changed please kindly contact me through **Zoom** or **Slack**, I'm available on Microverse UTC-6 time zone._

### CODE REVIEW

File: css/pop-up.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
+.pop-up {
+  scrollbar-gutter: stable;
+  padding: 4rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.6rem;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  width: 90%;
```
Comment (taldr27) [inline css/pop-up.css:10]: Great work on your popup for the mobile version! **For the desktop version, there are a couple of suggestions I'd like to make**:

- [ ] The size of the popup seems a **bit too big on desktop**. You could adjust it by removing or changing the width from **90%** to match the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=Lmbe0KVCHoQwSAR4-0). 🔍
- [ ] The space between the `X` and the top of the popup looks a bit larger than in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=Lmbe0KVCHoQwSAR4-0). To give your project a **more polished and professional look**, you could consider adjusting it to match the template. 😉

  ### Your design:
  image

  ### Figma design:
  image

File: javascripts/pop-up.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
+const projectDetails = [
+  {
+    name: 'Keeping track of hundreds of components #1',
+    deskcription: [
```
Comment (taldr27) [inline javascripts/pop-up.js:4]: - [ ] **[OPTIONAL]** You might want to fix this small typo in order to maintain **consistency and professionalism** in your project. 🙌

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello team @M-Anwar-Hussaini, @eamador1 , 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Populating the popup dynamically.
✅ Popup functionality working fine.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

File: javascripts/pop-up.js (hunk 2 above)
Comment (M-Anwar-Hussaini) [reply javascripts/pop-up.js:4]: I could not find this typo, kindly would you please mention where is it? @taldr27

Comment (taldr27) [reply javascripts/pop-up.js:4]: - [ ] **[OPTIONAL]** Sure, it's inside the `pop-up.js` file, make sure to change it in all the objects inside your `projectDetails` array. 🎯

  image

**Review by sumairq — APPROVED** [review]
### Hi @M-Anwar-Hussaini , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: javascripts/pop-up.js (hunk 2 above)
Comment (M-Anwar-Hussaini) [reply javascripts/pop-up.js:4]: Ok, Thank you very much. I'm going to solve it. 
You are a very passionate and kind code reviewer, I honestly appreciate your work.
@taldr27

**Pull request merged**